Documentation/Troubleshooting/BackendLogin.rst

```diff
@@ -0,0 +1,101 @@
+:navigation-title: Backend login
+..  include:: /Includes.rst.txt
+
+..  _troubleshooting-backend-login:
+
+===================================================
+Troubleshooting common TYPO3 backend login problems
+===================================================
+
+The following errors can happen during backend login:
+
+..  contents::
+
+..  _troubleshooting-login-reload:
+
+Login page reloads without error
+================================
+
+*   Try deleting all browser cookies and caches or use a different browser /
+    device. There might be an outdated cookie that prevents log-in.
+*   Clear all TYPO3 caches via console command or install tool.
+*   Delete folder :path:`var/cache` to remove possible corrupted cache data.
+*   Check the cookies configuration in TYPO3: Ensure that cookie-related
+    settings (:confval:`sessionTimeout <t3coreapi:globals-typo3-conf-vars-be-sessiontimeout>`,
+    :confval:`cookieDomain <t3coreapi:globals-typo3-conf-vars-be-cookiedomain>`)
```

Comment:
Should we mention the SYS cookieDomain here, too, as a fallback, if BE cookieDomain is not set?

Reply:
It is actually mentioned in the describtion of BE cookieDomain?

Reply:
Yes: "Same as [$TYPO3_CONF_VARS['SYS']['cookieDomain']](https://134sa.typo3retter.de/typo3/module/tools/settings#SYS-cookieDomain) but only for BE cookies. If empty, $TYPO3_CONF_VARS['SYS']['cookieDomain'] value will be used."